client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -69,13 +69,21 @@ class BusinessDetails extends Component {
 
 		const { getCurrencyConfig } = this.context;
 
-		const businessExtensions = Object.keys(
-			extensionInstallationOptions
-		).filter(
-			( key ) =>
-				extensionInstallationOptions[ key ] &&
-				key !== 'install_extensions'
-		);
+		const businessExtensions = Object.keys( extensionInstallationOptions )
+			.filter(
+				( key ) =>
+					extensionInstallationOptions[ key ] &&
+					key !== 'install_extensions'
+			)
+			.map( ( key ) => {
+				// Remove anything after :
+				// Please refer to selective-extensions-bundle/index.js
+				// installableExtensions variable
+				// this is to allow duplicate slugs (Tax & Shipping for example)
+				return key.split( ':' )[ 0 ];
+			} )
+			// remove duplicate
+			.filter( ( item, index, arr ) => arr.indexOf( item ) === index );
```

Comment:
This added complexity bugs me a bit, I think it could at least be unit tested to make sure it doesn't break in future.

if you export a function containing this logic then it will be easy to test the logic.

Reply:
@samueljseay Good idea 👍 I will add a new test for it.

Reply:
@samueljseay sorry for the delay in response. I've added a test in https://github.com/woocommerce/woocommerce-admin/pull/6547/commits/42cd68eb7303816514d5029e7cf1f1742c154ec4